.github/workflows/ci.yaml

```diff
@@ -20,22 +21,17 @@ jobs:
         run: npm run check
       - name: Build
         run: npm run build
-      - name: Upload pages artifact
-        uses: actions/upload-pages-artifact@v3
-        with:
-          path: dist
 
-  deploy:
-    needs: build
+ 
+  webhook-live:
```

Comment:
Might as well leave the old name, since it actually is deploying the site, only this time by calling a webhook instead of building and deploying it to pages.